server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/GetSnapshotsResponse.java

```diff
@@ -106,7 +82,9 @@ public int remaining() {
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         out.writeCollection(snapshots);
-        out.writeMap(failures, StreamOutput::writeException);
+        if (out.getTransportVersion().before(TransportVersions.REMOVE_SNAPSHOT_FAILURES)) {
+            out.writeMap(Map.of(), StreamOutput::writeException);
```

Comment:
Could you add a comment saying what this field used to be for?

Reply:
Added in [a748d09](https://github.com/elastic/elasticsearch/pull/114496/commits/a748d093ea6b22a75e9a676aec845c88d0f73446)